amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -134,22 +134,22 @@ enum Network {
 }
 
 enum DomainColor {
-  LIGHTPINK
+  LIGHT_PINK
```

Comment:
Changes those to have properly generated PascalCase members in CDapp

Reply:
You mean the underscores ?

Reply:
Yeah, if you don't include those, you get `Lightpink` instead of `LightPink`